Tests/LegacyUnit/FrontEnd/DefaultControllerTest.php

```diff
@@ -9452,6 +9453,161 @@ public function myEventsListForModifyMyEventsListRowHookWithoutInterfaceThrowsEx
+        $hook->expects(self::once())->method('modifyListFooter')->with($this->subject);
+        $hook->expects(self::once())->method('modifyEventBagBuilder')->with($this->subject, self::anything(), 'topic_list');
+        // We don't test for the second parameter (the bag builder instance here)
+        // because we cannot access it from the outside.
+        $hook->expects(self::never())->method('modifyRegistrationBagBuilder');
+
+        $hookClass = \get_class($hook);
+        $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['seminars'][SeminarListView::class][] = $hookClass;
+        GeneralUtility::addInstance($hookClass, $hook);
+
+        $this->subject->main('', []);
+    }
+
+    /**
+     * @test
+     */
+    public function listViewCallsSeminarListViewHookMethodsForSeminarList()
+    {
+        $this->subject->setConfigurationValue('what_to_display', 'seminar_list');
+
+        $hook = $this->createMock(SeminarListView::class);
+        $hook->expects(self::once())->method('modifyListHeader')->with($this->subject);
+        $hook->expects(self::once())->method('modifyListRow')->with($this->subject);
+        $hook->expects(self::never())->method('modifyMyEventsListRow');
+        $hook->expects(self::once())->method('modifyListFooter')->with($this->subject);
+        $hook->expects(self::once())->method('modifyEventBagBuilder')->with($this->subject, self::anything(), 'seminar_list');
+        // We don't test for the second parameter (the bag builder instance here)
+        // because we cannot access it from the outside.
+        $hook->expects(self::never())->method('modifyRegistrationBagBuilder');
+
+        $hookClass = \get_class($hook);
+        $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['seminars'][SeminarListView::class][] = $hookClass;
+        GeneralUtility::addInstance($hookClass, $hook);
+
+        $this->subject->main('', []);
+    }
+
+    /**
+     * @ test "events next day", "other dates" ?
+     */
+
+    /**
+     * @test
+     */
+    public function listViewCallsSeminarListViewHookMethodsForMyEventsList()
+    {
+        $this->subject->setConfigurationValue('what_to_display', 'my_events');
+
+        $this->createLogInAndRegisterFeUser();
+
+        $hook = $this->createMock(SeminarListView::class);
+        $hook->expects(self::once())->method('modifyListHeader')->with($this->subject);
+        $hook->expects(self::once())->method('modifyListRow')->with($this->subject);
+        $hook->expects(self::once())->method('modifyMyEventsListRow')->with($this->subject);
+        $hook->expects(self::once())->method('modifyListFooter')->with($this->subject);
+        $hook->expects(self::never())->method('modifyEventBagBuilder');
+        $hook->expects(self::once())->method('modifyRegistrationBagBuilder')->with($this->subject, self::anything(), 'my_events');
+        // We don't test for the second parameter (the bag builder instance here)
+        // because we cannot access it from the outside.
```

Comment:
This comment does not make sense here anymore.

Reply:
It is taken from Your tests that contain self::anything() instead of a real type. And You have them below the line they belong to. Put it before the line? Drop it?

Reply:
I don't remember why i put it there. (Usually comments should be above the referenced code, not below. Possibly it was a copy'n'paste error.) Please let's drop it.